client/components/site-selector/add-site.jsx

```diff
@@ -36,11 +34,7 @@ class SiteSelectorAddSite extends Component {
 	}
 
 	getAddNewSiteUrl() {
-		if ( this.props.hasJetpackSites ||
```

Comment:
Just to verify: previously we were displaying the new Jetpack page when the user has a Jetpack site or when running the AB test. This change will show the new "new site" page to all users, even to those who don't have a Jetpack site. Let's make sure that's intentional.